### PR TITLE
feat: add path label

### PR DESCRIPTION
**Pull request by rogerogers**
#### What type of PR is this?

feat: add path label

#### What this PR does / why we need it (English/Chinese):

en: add path label
zh: 增加路径标签

#### Which issue(s) this PR fixes:

#3

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -5,4 +5,5 @@ go 1.16
 require (
 	github.com/cloudwego/hertz v0.0.1
 	github.com/prometheus/client_golang v1.12.2
+	github.com/stretchr/testify v1.8.0
```
Comment (li-jin-gou) [inline go.mod:8]: add licenses

File: go.sum [hunk 2]
```diff
@@ -160,8 +160,10 @@ github.com/klauspost/cpuid/v2 v2.0.9/go.mod h1:FInQzS24/EEf25PyTYn52gqo7WaD8xa02
 github.com/konsorten/go-windows-terminal-sequences v1.0.1/go.mod h1:T0+1ngSBFLxvqU3pZ+m/2kptfBszLMUkC4ZK/EgS/cQ=
 github.com/konsorten/go-windows-terminal-sequences v1.0.3/go.mod h1:T0+1ngSBFLxvqU3pZ+m/2kptfBszLMUkC4ZK/EgS/cQ=
 github.com/kr/logfmt v0.0.0-20140226030751-b84e30acd515/go.mod h1:+0opPa2QZZtGFBFZlji/RkVcI2GknAs/DXo4wKdlNEc=
+github.com/kr/pretty v0.1.0 h1:L/CwN0zerZDmRFUapSPitk6f+Q3+0za1rQkzVuMiMFI=
 github.com/kr/pretty v0.1.0/go.mod h1:dAy3ld7l9f0ibDNOQOHHMYYIIbhfbHSm3C4ZsoJORNo=
 github.com/kr/pty v1.1.1/go.mod h1:pFQYn66WHrOpPYNljwOMqo10TkYh1fy3cYio2l3bCsQ=
+github.com/kr/text v0.1.0 h1:45sCR5RtlFHMR4UwH9sdQ5TC8v0qDQCHnXt+kaKSTVE=
```
Comment (li-jin-gou) [inline go.sum:168]: use `go mod tidy && go mod verify`

**Review by li-jin-gou — CHANGES_REQUESTED** [review]

File: go.sum (hunk 2 above)
Comment (rogerogers) [reply go.sum:168]: 我顺便更新到最新版本了。

File: go.mod (hunk 1 above)
Comment (rogerogers) [reply go.mod:8]: done

**Review by Duslia — APPROVED** [review]

**Review by li-jin-gou — APPROVED** [review]

**Pull request merged**